src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private final String firstLinePattern = "fruit,quantity" + System.lineSeparator();
+    private ReportService reportService;
+
+    @BeforeEach
+     void setUp() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    void emptyStorageValidReport_Ok() {
+        Storage.storage.clear();
```

Comment:
It's better to use AfterEach annotation to clear the storage. 
Check all places